TopMovies/Misc/Date+Extensions.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  Date+Extensions.swift
+//  TopMovies
+//
+//  Created by Macbook Pro  on 25.02.2021.
+//
+
+import Foundation
+
+extension Date {
+  static func prettyDate(_ dateString: String) -> String {
+    let dateFormatterGet = DateFormatter()
+    dateFormatterGet.dateFormat = "yyyy-mm-dd"
+    let dateFormatterPrint = DateFormatter()
+    dateFormatterPrint.dateFormat = "MMM, dd - yyyy"
```

Comment:
Learn a bit about the cost of `DateFormatter` initialization when it comes to performance:
http://jordansmith.io/performant-date-parsing/

https://developpaper.com/performance-optimization-exploration-of-dateformatters-mild-optimization/